scripts/ci.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+case "$1" in
+  "build")
+    yarn lint
+    yarn build
+    ./scripts/check-schema.sh
+    ;;
+  "test")
+    yarn jest test/ --collectCoverage=true
+    yarn codecov
+    yarn jest examples/
```

Comment:
Don't you have to run schema before example test?

Reply:
Don't we have the schema in the repo?